admin-dev/themes/new-theme/js/pages/product/services/combinations-service.js

```diff
@@ -74,4 +92,27 @@ export default class CombinationsService {
       productId: this.productId,
     }));
   }
+
+  /**
+   * @param {string} orderBy
+   * @param {string} orderWay
+   */
+  setOrderBy(orderBy, orderWay) {
+    this.orderBy = orderBy;
+    this.orderWay = orderWay.toLowerCase() === 'desc' ? 'DESC' : 'ASC';
```

Comment:
is this really necessary?
maybe `this.orderWay = orderWay` would be enough? I don't think that endpoint cares whether its uppercase or not. (sql doesnt care for sure).
Unless it is a safeguard to avoid any invalid values :man_shrugging:? - (though i doubt its necessary too)

Reply:
Yeah it's more a habit of safe guarding, because you're never really sure of the implementation If the value is only used and injected in the SQL it's fine But sometimes it's interpreted and compared so usually I always perform check (in BO and in FO) based on the lower case to avoid any incompatibility

Besides injecting the user input directly might lead to SQL injection, so I guess it's always preferable to check them in server side